scss/application.scss

```diff
@@ -164,13 +137,50 @@ body {
       height: $triangle-height;
     }
   }
+
+  &-content {
+    width: 100%;
+    max-width: 1200px;
+    display: flex;
+    margin: 60px auto;
+    flex-flow: row wrap;
+    justify-content: flex-start;
+    display: grid;
+    align-content: space-evenly;
+    justify-content: space-evenly;
+    grid-template-columns: $tech-card-width $tech-card-width $tech-card-width;
+    grid-template-rows: $tech-card-height $tech-card-height
```

Comment:
There's no need to set both properties. Yo can just use `grid-template-columns` and that will work. Also, instead of repeating the `$tech-card-width`, you can set it to `repeat(3, max-content)` and will be the same :wink: